### PR TITLE
Added online.net to hosting

### PR DESCRIPTION
**Pull request by alpha14**
- Added entry and icon

I couldn't find any doc url.

### CODE REVIEW

**Comment by mxxcon** [comment]
How do you know it supports sms and software?

**Comment by alpha14** [comment]
from the account's control panel:
screenshot_2015-07-28_19-41-43

File: _data/hosting.yml [hunk 1]
```diff
@@ -100,6 +100,14 @@ websites:
       software: Yes
       doc: https://blog.nearlyfreespeech.net/2014/02/28/price-cuts-more-security-and-recovery-options/
 
+    - name: Online
+      url: https://www.online.net/
+      img: online.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      twitter: online_en
```
Comment (Carlgo11) [inline _data/hosting.yml]: they support tfa right? Then you don't need a twitter handle. For more info see [our wiki](https://github.com/jdavis/twofactorauth/wiki/FAQ_New_TFA_SITE)

**Comment by jamcat22** [comment]
@mxxcon @Carlgo11 **ahem**

**Pull request merged**